production/ksonnet/README.md

```diff
@@ -27,7 +30,7 @@ Grab the promtail module using jb:
 $ jb install github.com/grafana/loki/production/ksonnet/promtail
 ```
 
-Replace the contents of `environments/loki/main.jsonnet` with:
+Replace the contents of `environments/default/main.jsonnet` with:
```

Comment:
An environment is created above is called `loki`, no need to use the default environment here